exercises/practice/leap/.meta/config.json

```diff
@@ -19,5 +19,5 @@
   },
   "blurb": "Given a year, report if it is a leap year.",
   "source": "JavaRanch Cattle Drive, exercise 3",
-  "source_url": "https://coderanch.com/t/718816/Leap"
+  "source_url": "http://www.javaranch.com/leap.jsp"
```

Comment:
@kytrinyx A lot of these links should be updated to use https in the problem specifications. We had to fix them in order for our Links action to work. It's disabled temporarily due to an unrelated issue.

Reply:
Thanks for the heads up; I'll submit a PR to the problem-specifications repository.

Reply:
Thanks